pkg/kubectl/polymorphichelpers/helpers.go

```diff
@@ -233,3 +276,162 @@ func SelectorsForObject(object runtime.Object) (namespace string, selector label
 
 	return namespace, selector, nil
 }
+
+// Generators returns the generators for the provided command
+func Generators(cmdName string) map[string]kubectl.Generator {
```

Comment:
No, these shouldn't live here.  Generators have been a disaster, but this isn't how to fix them.